src/compiler/crystal/config.cr

```diff
@@ -16,11 +16,14 @@ module Crystal
 
     def self.description
       formatted_sha = "[#{build_commit}] " if build_commit
+      formatted_date = "(#{date})" unless date.empty?
+
       <<-DOC
-        Crystal #{version} #{formatted_sha}(#{date})
+        Crystal #{version} #{formatted_sha}#{formatted_date}
```

Comment:
If a date isn't available, the description would have a trailing space on this line

Reply:
Good catch, I updated it to use Enumerable#join